ecolex/legislation.py

```diff
@@ -129,25 +119,40 @@ def harvest_file(upfile):
             key = v['en2'].lower()
             all_languages[key] = v
 
+    resp = requests.get('http://extwprlegs1.fao.org/cgi-bin/xml.exe?database=valid&' \
+                        'search_type=query&table=all&query=T:desext&format_name=@XECOLEX&' \
+                        'lang=xmlf&page_header=@EXMLH&page_footer=@EXMLF&sort_name=@SNAME')
```

Comment:
Don't hardcode this URL. Better options: add it to `settings.py` / receive it as a parameter on the management command.